libazureinit/src/error.rs

```diff
@@ -29,4 +29,8 @@ pub enum Error {
     UserMissing { user: String },
     #[error("Provisioning a user with a non-empty password is not supported")]
     NonEmptyPassword,
+    #[error("Unable to get list of block devices")]
+    BlockUtils(#[from] block_utils::BlockUtilsError),
+    #[error("Unable to get list of block devices")]
+    BlockUtilsError,
```

Comment:
I'd recommend dropping this and using `anyhow::anyhow!` in the application. The library never returns this error so it would be confusing to a library user to have to consider how to handle it when it never occurs.

Reply:
Done